spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -699,14 +699,14 @@ for _, strategy in helpers.each_strategy() do
           helpers.wait_until(function()
             local proxy_client = helpers.http_client("127.0.0.1", 9002)
             -- serviceless route should return 503 instead of 404
-            res = proxy_client:get("/2")
+            res = proxy_client:get("/5")
```

Comment:
I'm not very sure about the purpose of this test. It looks like that, it wants to make sure when 2 is synced, all changes should be synced(2 will not be synced individually).
Or, should this test is checking when a change takes effect, wether other changes before that works?